qiskit_sphinx_theme/furo/base/partials/icons.html

```diff
@@ -135,5 +135,19 @@
             </g>
         </g>
     </svg>
+  </symbol>
+    {#- QISKIT CHANGE: end. -#}
+    {#- QISKIT CHANGE: start. -#}
+  <symbol id="svg-qiskit-chevron-left" viewBox="0 0 24 24">
```

Comment:
Rather than defining a new symbol, we will want to use `svg-arrow-right` from https://github.com/Qiskit/qiskit_sphinx_theme/pull/395, and then use CSS to rotate the arrow, like this:

https://github.com/Qiskit/qiskit_sphinx_theme/blob/49d9e1f8f0aa5af5bcca02802cd8c97e785ddd4b/qiskit_sphinx_theme/furo/base/static/styles/qiskit_changes.css#L226-L228

That results in less complex code.